data/abilities.js

```diff
@@ -4658,7 +4658,7 @@ let BattleAbilities = {
 	wonderguard: {
 		shortDesc: "This Pokemon can only be damaged by supereffective moves and indirect damage.",
 		onTryHit(target, source, move) {
-			if (target === source || move.category === 'Status' || move.type === '???' || move.id === 'struggle') return;
+			if (target === source || move.category === 'Status' || move.type === '???' || move.id === 'struggle' || move.id === 'skydrop') return;
```

Comment:
```suggestion
			if (target === source || move.category === 'Status' || move.type === '???' || move.id === 'struggle') return;
			if (move.id === 'skydrop' && !target.volatiles['skydrop']) return;
```

Reply:
If you check `!source.volatiles['skydrop']` rather than `target`, this suggestion works.